From: Jiri Slaby <jslaby@suse.cz>
Date: Thu, 16 May 2024 11:00:36 +0200
Subject: ioctls/kvm: make s390 ioctls s390-only
Git-repo: https://github.com/jirislaby/trinity
Git-commit: 44013fd01a1e857d5bcb79b80fa4c8656d43bde2
Patch-mainline: submitted, https://github.com/kernelslacker/trinity/pull/48
References: kernel 6.8 & 6.9

They are now s390-only since commit 71cd774ad2f9 (KVM: s390: move
s390-specific structs to uapi/asm/kvm.h).

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 ioctls/kvm.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ioctls/kvm.c b/ioctls/kvm.c
index 30aeed7a..8a2ef713 100644
--- a/ioctls/kvm.c
+++ b/ioctls/kvm.c
@@ -15,8 +15,10 @@ static const struct ioctl kvm_ioctls[] = {
 	IOCTL(KVM_SET_USER_MEMORY_REGION),
 	IOCTL(KVM_SET_TSS_ADDR),
 	IOCTL(KVM_SET_IDENTITY_MAP_ADDR),
+#if defined(__s390__)
 	IOCTL(KVM_S390_UCAS_MAP),
 	IOCTL(KVM_S390_UCAS_UNMAP),
+#endif
 	IOCTL(KVM_S390_VCPU_FAULT),
 	IOCTL(KVM_CREATE_IRQCHIP),
 	IOCTL(KVM_IRQ_LINE),
-- 
2.45.0

